nomenclature/definitions/region/subcountries.yaml

```
@@ -275,33 +719,81 @@ Italy|Subgrid south:
 
 # ehighway north sea clusters
 Belgium|North Sea:
-    definition: this 'virtual' region holds the offshore windpower of belgium in norh sea
+    definition: This 'virtual' region holds the offshore windpower of belgium in norh sea
```

There is a typo here (and in all similar subgrid definitions, and proposing a slight rewording
```suggestion
    definition: This 'virtual' region represents offshore wind generation of Belgium in the North Sea
```